src/utilities/api.ts

```diff
@@ -1,93 +1,65 @@
+import { TopComics } from "../models/homePageData";
+import { IComicData } from "../models/comicData";
+import { IComicChapters, ISingularChapterMeta } from "../models/chapterData";
+import { IGenreData } from "../models/genreData";
+
 // get top manga
-export function getTrendingManga() {
-  fetch(
+export function getTrendingManga(): Promise<TopComics> {
+  return fetch(
     "https://api.comick.io/top?type=trending&comic_types=manga&accept_mature_content=false"
```

Comment:
I recommend having the api base URL as a variable. That way, if it somehow changes, it won't be difficult to change for every query. Just a consideration.

Reply:
true, and it be changing a lot. Done, thank you